projects/cm_mcu/ZynqMonTask.c

```diff
@@ -528,9 +548,9 @@ void zm_send_data(struct zynqmon_data_t data[])
 {
   // https://docs.google.com/spreadsheets/d/1E-JD7sRUnkbXNqfgCUgriTZWfCXark6IN9ir_9b362M/edit#gid=0
   for (int i = 0; i < ZMON_VALID_ENTRIES; ++i) {
-    uint8_t message[4];
+    uint8_t message[5];
```

Comment:
maybe introduce some #define constant for the size of the message

Reply:
Sounds good, will do. 
Done.